functions/validator/requirements.txt

```diff
@@ -14,3 +14,5 @@ packaging==16.8
 setuptools==35.0.2
 appdirs==1.4.3
 pyparsing==2.2.0
+datetime
+git+git://github.com/mozilla-iam/cis.git
```

Comment:
actually boto3 is not missing.  This is a dev requirement only for running outside lambda.  Should not be bundled at deploy time as boto3 is in lambda by default.